docs/distributions/fedora/installation.md

```diff
@@ -11,13 +11,12 @@ If you need a more updated kernel, use the iso from [sharpenedblade](https://git
 # Install Procedure
 
 1. Follow the [Pre-Install](https://wiki.t2linux.org/guides/preinstall) guide.
-2. Use the installer like normal until you get to the partitioning step.
-3. Click on automatic partitioning, then click *create free space*
-4. Delete the partition you created using macOS for Linux.
-5. Continue with the rest of the installation.
-6. Once it's finished, you can reboot without your installation media. Hold down Option (⌥) while booting, then select EFI Boot and press enter.
-7. Welcome to Fedora! :)
-8. Once you're booted and in your desktop, set up [Wi-Fi](https://wiki.t2linux.org/guides/wifi-bluetooth/) to finish the installation.
+2. Click on *Share disk with other operating system*, then select *Reclaim additional space*.
+3. Delete the partition you created using macOS for Linux.
+4. Continue with the rest of the installation.
+5. Once it's finished, you can reboot without your installation media. Hold down Option (⌥) while booting, then select EFI Boot and press enter.
+6. Welcome to Fedora! :)
+7. Once you're booted and in your desktop, set up [Wi-Fi](https://wiki.t2linux.org/guides/wifi-bluetooth/) to finish the installation.
```

Comment:
If you have set up the systemd service, then probably say that if wifi should work after first boot, but if it is not working, then follow this guide.